tests/test_files.py

```diff
@@ -557,18 +542,15 @@ def test_convertible_substitles_from_pressurcooker(pressurcooker_test_files):
             ), "missing check_words in converted subs"
 
 
-def test_convertible_substitles_ar_ttml(youtube_test_file):
+def test_convertible_substitles_ar_ttml():
     """
     Regression test to make sure correct lang_code is detected from .ttml data.
     """
-    # local_path = os.path.join(
-    #     "tests", "testcontent", "downloaded", "testsubtitles_ar.ttml"
-    # )
     local_path = os.path.abspath(
         os.path.join(
             os.path.dirname(__file__),
             "testcontent",
-            "downloaded",
+            "samples",
```

Comment:
is this changed because we've moved the file path to be under /samples so that it's clearer that it's... a sample? but content-wise it's the same?

Reply:
Yes - it was originally setup to download, but the download kept failing so it was committed to the repo, but never moved to a folder where that should happen (as opposed to a folder that gets cleaned up by our `make clean` command).